docs/how-to/HA.md

```diff
@@ -0,0 +1,681 @@
+# Set up PMM in HA mode
+
+!!! caution alert alert-warning "Important"
+    This feature is still in [Technical Preview](https://docs.percona.com/percona-monitoring-and-management/details/glossary.html#technical-preview) and is subject to change. We recommend that early adopters use this feature for testing purposes only.
+
+
+Set up PMM using Docker containers in a high-availability (HA) configuration following these instructions. 
```

Comment:
Can we mention here or in the previous alert-warning that the HA mode has been added from version 2.41? so that people will understand the min required version to run in HA mode. Thanks